Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -66,6 +72,48 @@ class NotificationsViewController: UIViewController {
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
     }
+
+    private func setupEmptyStateView() {
+        view.addSubview(emptyStateView)
+
+        emptyStateView.snp.makeConstraints { make in
+            make.top.bottom.leading.trailing.equalToSuperview()
+        }
+
+        let notifBellImageView = UIImageView()
+        notifBellImageView.image = UIImage.notifIcon
```

Comment:
Can we make this `UIImage.scooped.notifIcon` pls lol (yes this means everywhere else should be using this). You want to be able to distinguish whether it's a built in UIImage versus a Scooped one.